x/ibctransferhooks/types/hooks.go

```diff
@@ -0,0 +1,34 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	"github.com/cosmos/ibc-go/v3/modules/core/exported"
+)
+
+var _ IBCTransferHooks = MultiIBCTransferHooks{}
+
+// combine multiple ibc transfer hooks, all hook functions are run in array sequence
```

Comment:
```suggestion
// MultiIBCTransferHooks combine multiple ibc transfer hooks, all hook functions are run in array sequence
```

Reply:
Will take this manually or can merge this at the final stage of merging this PR